frontend/src/components/Step4.tsx

```diff
@@ -0,0 +1,154 @@
+
+              <label htmlFor="licenseNumber" className={styles.label}>
+                Certified/Registered/License Number
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="licenseNumber"
+                  id="licenseNumber"
+                  placeholder="Enter Certified/Registered/License Number"
+                />
+              </label>
+            </div>
+
+            <div className={styles.formRow}>
+              <label htmlFor="state" className={styles.label}>
+                Name of State/Jurisdiction
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="state"
+                  id="state"
+                  placeholder="Enter Name of State/Jurisdiction"
+                  pattern="Alabama|Alaska|Arizona|Arkansas|California|Colorado|Connecticut|Delaware|Florida|Georgia|Hawaii|Idaho|Illinois|Indiana|Iowa|Kansas|Kentucky|Louisiana|Maine|Maryland|Massachusetts|Michigan|Minnesota|Mississippi|Missouri|Montana|Nebraska|Nevada|New Hampshire|New Jersey|New Mexico|New York|North Carolina|North Dakota|Ohio|Oklahoma|Oregon|Pennsylvania|Rhode Island|South Carolina|South Dakota|Tennessee|Texas|Utah|Vermont|Virginia|Washington|West Virginia|Wisconsin|Wyoming"
+                />
+              </label>
+
+              <label htmlFor="licenseExpiration" className={styles.label}>
+                Date Certified/Registered/License Expires
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="licenseExpiration"
+                  id="licenseExpiration"
+                  placeholder="Enter Date Certified/Registered/License Expires"
+                />
+              </label>
+            </div>
+
+            <div className={styles.formRow}>
+              <label htmlFor="certificationExam" className={styles.label}>
+                Certification Exam
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="certificationExam"
+                  id="certificationExam"
+                  placeholder="Enter Certification Exam"
+                />
+              </label>
+
+              <label htmlFor="anwser" className={styles.label}>
+                Have you ever been convicted of a Felony?
+                <Dropdown options={["Yes", "No"]}></Dropdown>
+              </label>
+            </div>
+
+            <div className={styles.formRow}>
+              <label htmlFor="dateOfCertificationExam" className={styles.label}>
+                Date of Certification Exam
+                <input
+                  className={styles.input}
+                  type="text"
+                  name="dateOfCertificationExam"
+                  placeholder="mm/dd/yyyy"
+                  id="dateOfCertificationExam"
+                  pattern="\d{2}-\d{2}-\d{4}"
```

Comment:
the "-" should be "/" the template asks for slashes but you are checking for dashes.